pkg/admin/prerun/upgrade_blocking.go

```diff
@@ -5,26 +5,37 @@ import (
 	"errors"
 	"fmt"
 	"io/fs"
+	"strings"
 
 	embedded "github.com/openshift/microshift/assets"
 	"k8s.io/klog/v2"
 )
 
 func isUpgradeBlocked(execVersion versionMetadata, dataVersion versionMetadata) error {
+	klog.InfoS("START obtaining list of blocked upgrades")
```

Comment:
I wonder each step here really needs to be a separate "unit of work". The whole function might be 1 unit doing the work of "checking upgrade compatibility" or something, from the user's perspective. If any of the internal steps is relatively slow, that could indicate that step should be a separate unit, but I think they're all pretty fast, right?

Let's keep this, but think more about where we draw the line about what a "unit" is and see if we can find the right balance.

Reply:
I'm wondering  that myself, but given time constraints and feature being young, I'd rather have too verbose logs than not enough of them (and it's only about 60 lines)